wordpress-mobile/gutenberg-mobile
# Revert - add selectionStart and selectionEnd to transientEdits

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/2349
This PR reverts commit that introduced regression in the client app.

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/22933

To test:
- Run WP-iOS app
- type something really fast and press publish
- the correct version should be published

- type something in the paragraph
- the undo level should be created after each sign

PR submission checklist:
